src/backend/gl/src/device.rs

```diff
@@ -1402,7 +1402,11 @@ impl d::Device<B> for Device {
     ) -> Result<n::Image, i::CreationError> {
         let gl = &self.share.context;
 
-        let desc = conv::describe_format(format).unwrap();
+        let desc;
+        match conv::describe_format(format) {
```

Comment:
let's do it with expressions:
```rust
let desc = conv::describe_format(format).ok_or(i::CreationError::Format(format))?;
```

Reply:
Yes, much better. Learning better Rust in the way!